src/sentry/api/endpoints/organization_code_mappings.py

```diff
@@ -126,36 +126,33 @@ def get(self, request: Request, organization) -> Response:
 
         :pparam string organization_slug: the slug of the organization the
                                           team should be created for.
-        :queryparam int integrationId: the optional integration id.
-        :queryparam int projectId: the optional project id.
+        :qparam int integrationId: the optional integration id.
+        :qparam int projectId: Optional. Pass "-1" to filter to 'all projects user has access to'. Omit to filter for 'all projects user is a member of'.
+        :qparam int per_page: Pagination size.
+        :qparam string cursor: Pagination cursor.
         :auth: required
         """
 
         integration_id = request.GET.get("integrationId")
-        project_id = request.GET.get("projectId")
-
-        if not integration_id and not project_id:
-            return self.respond(
-                {"detail": 'Missing valid "projectId" or "integrationId"'},
-                status=status.HTTP_400_BAD_REQUEST,
-            )
 
         queryset = RepositoryProjectPathConfig.objects.all()
 
         if integration_id:
             # get_organization_integration will raise a 404 if no org_integration is found
             org_integration = self.get_organization_integration(organization, integration_id)
             queryset = queryset.filter(organization_integration=org_integration)
-
-        if project_id:
-            # Check that the project is apart of the organization. get_project will raise 404 if project not found.
-            project = self.get_project(organization, project_id)
-            queryset = queryset.filter(project=project)
-
-        # front end handles ordering
-        # TODO: Add pagination
-        data = map(lambda x: serialize(x, request.user), queryset)
-        return self.respond(data)
+        else:
+            # Filter by project
+            project_ids = self.get_requested_project_ids_unchecked(request, "projectId")
```

Comment:
Instead of passing `projectId` as a parameter to `get_requested_project_ids_unchecked()`, you should modify the frontend to pass the id as `project`.